### PR TITLE
[INTERNAL] Fixes man page output: Escape special characters '.' or ''' at the beginning of a line.

### LINKED ISSUE #2279: unix manual page generation errors
<pre>
W: seqan-apps: manpage-has-errors-from-man usr/share/man/man1/micro_razers.1.gz 47: warning: macro `N'' not defined
N: 
N:    This man page provokes warnings or errors from man.
[…]
N:    Other warnings are often formatting typos, like missing quotes around a
N:    string argument to .IP. These are likely to result in lost or malformed
N:    output. See the groff_man (or groff_mdoc if using mdoc) man page for
N:    information on macros.
N:    
N:    This test uses man's --warnings option to enable groff warnings that
N:    catch common mistakes, such as putting . or ' characters at the start of
N:    a line when they are intended as literal text rather than groff
N:    commands. This can be fixed either by reformatting the paragraph so that
N:    these characters are not at the start of a line, or by adding a
N:    zero-width space (\&) immediately before them.
N:    
N:    At worst, warning messages can be disabled with the .warn directive, see
N:    "Debugging" in the groff manual.
N:    
N:    Lintian also stricter in regards to declaring manpage preprocessors.
N:    
N:    To test this for yourself you can use the following command:
N:     LC_ALL=en_US.UTF-8 MANROFFSEQ='' MANWIDTH=80 \
N:            man --warnings -E UTF-8 -l -Tutf8 -Z <file> >/dev/null
N:    
N:    Refer to the groff_man(7) manual page and the groff_mdoc(7) manual page
N:    for details.
N:    
N:    Severity: normal, Certainty: certain
N:    
N:    Check: manpages, Type: binary
N: 
W: seqan-apps: manpage-has-errors-from-man usr/share/man/man1/razers.1.gz 128: warning: macro `razers' not defined (possibly missing space after `ra')
W: seqan-apps: manpage-has-errors-from-man usr/share/man/man1/razers3.1.gz 167: warning: macro `razers' not defined (possibly missing space after `ra')
N: 1 tag overridden (1 error)
</pre>

### PR DESCRIPTION
**Pull request by smehringer**
Fixes #2279

### CODE REVIEW

File: include/seqan/arg_parse/tool_doc.h [hunk 1]
```diff
@@ -1531,6 +1531,20 @@ void TextToolDocPrinter_::print(std::ostream & stream, ToolDoc const & doc)
     }
 }
 
+// --------------------------------------------------------------------------
+// Function ManToolDocPrinter_::print()
+// --------------------------------------------------------------------------
+
+//brief: '.' or ''' indicate man macros that need to be escaped wiht \\&
```
Comment (rrahn) [inline include/seqan/arg_parse/tool_doc.h]: typos here.

**Comment by smehringer** [comment]
What keeps this from being merged ?

**Comment by h-2** [comment]
the windows error is unrelated?

**Comment by smehringer** [comment]
I cannot not even access the link to Jenkins any more. How do I rebuild again?
But yes I think the build error was unrelated.

**Comment by h-2** [comment]
!rebuild

**Comment by smehringer** [comment]
can you click on the `Details` link? I get an 404 error..

**Pull request merged**